tests/python/contrib/test_hexagon/conv2d/test_conv2d_blocked.py

```diff
@@ -164,11 +163,11 @@ def test_conv2d(
         k_split_factor,
         h_split_factor,
         dtype,
-        target,
     ):
         """conv2d test"""
         # TODO: no support for dilation
         dilation = 1
+        target = "llvm"
```

Comment:
Same question here, no hexagon hardware in use for tests in this file.

Reply:
reverted it